pages/common/dbt.md

```diff
@@ -0,0 +1,32 @@
+# dbt
+
+> A tool to model transformations in data warehouses.
+> More information: <https://github.com/dbt-labs/dbt-core>.
+
+- Debug the dbt project and the connection to the database:
+
+`dbt debug`
+
+- Run all models of the project:
+
+`dbt run`
+
+- Run all tests of `example_model`:
+
+`dbt test --select example_model`
+
+- Build (load seeds, run models, snapshots, and tests associated with) `example_model` and its downstream dependents:
+
+`dbt build --select example_model+`
```

Comment:
```suggestion
`dbt build --select example_model`
```
Is the `+` a typo?

Reply:
No, it's the syntax for  selecting for all downstream dependents of `example_model`.